content/en/docs/contribute/participate/issue-wrangler.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Issue Wranglers
+content_type: concept
+weight: 20
+---
+
+<!-- overview -->
+
+In order to reduce the burden on the [PR Wrangler](/docs/contribute/participate/pr-wrangler), formal approvers, and reviewers, members of SIG Docs take week long shifts [triaging and categorising issues](/docs/contribute/review/for-approvers.md/#triage-and-categorize-issues) for the repository.
+
+<!-- body -->
+
+## Duties
+
+Each day in a week-long shift as Issue Wrangler:
+
+- Triage and tag incoming issues daily. See [Triage and categorize issues](https://github.com/kubernetes/website/blob/main/docs/contribute/review/for-approvers/#triage-and-categorize-issues) for guidelines on how SIG Docs uses metadata.
+- Identifying whether the issue falls under the support category and assigning a "triage/accepted" status.
+- Assuring the issue is tagged with the appropriate sig/area/kind labels.
+- Keeping an eye on stale & rotten issues within the kubernetes/website repository.
+- [Issues board](https://github.com/orgs/kubernetes/projects/72/views/1) maintenance would be nice
+
+### Requirements
+
+- Must be an active member of the Kubernetes organization.
+- A minimum of 15 quality contributions to Kubernetes (of which a certain amount should be directed towards kubernetes/website).
```

Comment:
@natalisucks @reylejano any specific number that you want to add here for the bracketed quote?
```suggestion
- A minimum of 15 quality contributions to Kubernetes (of which a certain amount should be directed towards kubernetes/website).
```

Reply:
Can we replace the word "quality" with "non-trivial" and then link to the trivial edits policy? I think that would be clearer for people in terms of what we mean by quality. WDYT?

Reply:
+1

Reply:
@natalisucks can you please share with me the link to the page that needs to be added?

Reply:
_Trivial Edits link: https://www.kubernetes.dev/docs/guide/pull-requests/#trivial-edits_